sdk/storage/azure-storage-blob/src/test/java/com/azure/storage/blob/AppendBlobAPITest.groovy

```diff
@@ -325,6 +325,8 @@ class AppendBlobAPITest extends APISpec {
         ByteArrayOutputStream downloadStream = new ByteArrayOutputStream(1024)
         destURL.download(downloadStream)
         downloadStream.toByteArray() == Arrays.copyOfRange(data, 2 * 1024, 3 * 1024)
+        cleanup:
+        defaultInputStream.get().reset()
```

Comment:
`defaultInputStream` is a `Supplier<InputStream>` that news up a stream of the bytes of a preset string and just gives us that. These cleanup blocks are just getting a brand new stream only to call `reset()` and discard the reference for the GC to pick up.

Reply:
Revert it back.